docs/datasets_and_tables/gtfs_realtime.md

```diff
@@ -0,0 +1,58 @@
+(gtfs-realtime)=
+# GTFS Realtime
+
+We collect realtime data every 20 seconds for feeds listed in [agencies.yml](../airflow/agencies.md).
+This data is processed and validated daily.
+
+## How to view the `GTFS Realtime` documentation
+
+Visit the [dbt Cal-ITP warehouse documentation](https://dbt-docs.calitp.org/#!/overview).
+
+For help navgating the documentation, visit [Navigating the dbt Docs](navigating-dbt-docs).
+
+## Warehouse Schemas
+
+### Source
+| dataset name | description |
+| ------- | ----------- |
+| `external_gtfs_rt` | Hive-partitioned external tables reading GTFS RT data and validation errors from GCS. |
+| `gtfs_rt_logs` | Data in the gtfs_rt_logs dataset in BigQuery, from logs sink. |
+| `gtfs_rt` | Data in the gtfs_rt dataset in BigQuery, generally produced by the rt_loader and rt_loader_files Airflow DAGs. |
+
+### Staging
+| dataset name | description |
+| ------- | ----------- |
+| `staging` | tables with a prefix `staging.stg_rt__*` |
+
+### Internal Tables
```

Comment:
Can we clarify what is meant by "internal" here? I'm not really sure what most of these are referring to... Wonder if this section can actually be deleted?

Reply:
@lauriemerrell that already existed, I just kept it. happy to delete...

Reply:
Yeah,  I think deletion is cleaner ✅